### PR TITLE
watcher: work around trace bug in status-wd

### PR DESCRIPTION
**Pull request by sam-github**

### CODE REVIEW

File: lib/watcher/status-wd.js [hunk 1]
```diff
@@ -29,3 +29,49 @@ exports.master = function(handle) {
...
+    pkgjson = require(cwd + '/package.json');
+  } catch (e) {
+    pkgjson = {};
+  }
+  try {
+    userjson = {} // require(home + '/strongloop.json');
+  } catch (e) {
+    userjson = {};
+  }
+  console.log('REQUIRE', require);
```
Comment (kraman) [inline lib/watcher/status-wd.js]: missed a console.log :)

File: lib/watcher/status-wd.js [hunk 2]
```diff
@@ -29,3 +29,49 @@ exports.master = function(handle) {
...
+  try {
+    pkgjson = require(cwd + '/package.json');
+  } catch (e) {
+    pkgjson = {};
+  }
+  try {
+    userjson = {} // require(home + '/strongloop.json');
+  } catch (e) {
+    userjson = {};
+  }
```
Comment (kraman) [inline lib/watcher/status-wd.js]: no-op, can just remove the try/catch here

File: lib/watcher/status-wd.js [hunk 3]
```diff
@@ -29,3 +29,49 @@ exports.master = function(handle) {
...
+    appName: appName || env.STRONGLOOP_APPNAME || env.SL_APP_NAME ||
+                 nfjson.appName || pkgjson.name || userjson.appName,
+  };
+
+  appName = config.appName;
+
+  if (appName instanceof Array) {
+    config.appName = appName.shift();
+  } else {
+  }
```
Comment (kraman) [inline lib/watcher/status-wd.js]: no-op

File: lib/watcher/status-wd.js (hunk 2 above)
Comment (sam-github) [inline lib/watcher/status-wd.js]: I don't want to change anything, this is an almost literal copy of the original source

File: lib/watcher/status-wd.js (hunk 3 above)
Comment (sam-github) [inline lib/watcher/status-wd.js]: ditto

**Comment by kraman** [comment]
Trying this fix to see if it works

**Comment by kraman** [comment]
LGTM

**Pull request merged**